docs/web/docs/guides/tutorials/first_task.md

```diff
@@ -236,45 +236,54 @@ Complete this task, and you can review it in the same way as before.
 from mephisto.abstractions.blueprints.static_html_task.static_html_blueprint import (
-    BLUEPRINT_TYPE,
+    BLUEPRINT_TYPE_STATIC_HTML,
+)
+from mephisto.tools.scripts import process_config_and_get_operator, task_script
+from omegaconf import DictConfig
+
+@task_script(default_config_file="example")
+def main(operator: Operator, cfg: DictConfig) -> None:
+```
+This is all you really *need* to launch a Mephisto task! The `@task_script` decorator does the job of attaching your hydra yaml as default arguments for the main.
+
+Of course, there's quite a bit of 'magic' happening underneath the hood thanks to the script utilities. This version is explicit to show where you may add customization, and re-ordered for understanding:
+```python
+# modified static_test_script.py
+from mephisto.abstractions.blueprints.static_html_task.static_html_blueprint import (
+    BLUEPRINT_TYPE_STATIC_HTML,
 )
-...
 
-defaults = ["_self_", {"conf": "example"}]
+from mephisto.tools.scripts import task_script
+from mephisto.operations.hydra_config import build_default_task_config
 
-from mephisto.operations.hydra_config import RunScriptConfig, register_script_config
-TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
 @dataclass
-class TestScriptConfig(RunScriptConfig):
-    defaults: List[Any] = field(default_factory=lambda: defaults)
-    task_dir: str = TASK_DIRECTORY
+class MyTaskConfig(build_default_task_config('example')):
+    custom_args: Any = 4
 
-register_script_config(name="scriptconfig", module=TestScriptConfig)
-@hydra.main(config_path="hydra_configs", config_name="scriptconfig")
+@task_script(config=MyTaskConfig)
+def main(operator: Operator, cfg: DictConfig) -> None:
 ```
 In this snippet, we do a few things:
-1. We import the `BLUEPRINT_TYPE` to force an import of the blueprint we intend to use with this run script. This isn't *required* for Mephisto default blueprints, but is important for custom `Blueprint`s to ensure they are loaded into the Mephisto registry.
-2. We set up the default `conf` file to be `example`, which ensures that Hydra looks into `conf/example.yaml` on any run where this isn't overridden.
-3. We create a `RunScriptConfig`, which is a set of options that we are allowed to set for a task. Here, we only provide a `task_dir`, which you can override on the command line with `python static_test_script.py task_dir=some/other/path`. Mostly, we use this for ensuring that the `data_csv` and other path-related variables are portable.
-4. We register this config with hydra, and then have the `main` function point to this configuration. Note the `hydra_configs` value for `config_path`, which now fully explains the `hydra_configs/conf/example.yaml` path. 
+1. We import the `BLUEPRINT_TYPE_STATIC_HTML` to force an import of the blueprint we intend to use with this run script. This isn't *required* for Mephisto default blueprints, but is important for custom `Blueprint`s to ensure they are loaded into the Mephisto registry.
```

Comment:
A little confused by this - why is the import actually necessary? Does the import produce side-effects? Won't it get triggered at some point by the underlying library (I'm assuming Mephisto will import the Blueprint mentioned via Hydra), if not by the user directly here?

"Not required... but important" leaves me a little confused about the implications of either decision.

Reply:
Ah okay it's not required for _anything_ in the Mephisto library, but that's because we already know where they are and can trigger adding to the registry. For things _outside_ the mephisto library, this is required. I'll update.